Formula/broot.rb

```diff
@@ -16,6 +16,7 @@ class Broot < Formula
 
   depends_on "rust" => :build
 
+  uses_from_macos "curl" => :build
```

Comment:
Can someone check if this actually does something different on Linux?
- `curl` is `keg_only :provided_by_macos` so I assume it is linked on Linux.
- `rust` is `uses_from_macos "curl"` so the libraries should be compiled with brewed curl and brewed `curl` will be installed via dependency chain.

Not sure if cargo command uses actual `curl` command. I would have expected curl to be compiled and the API to be directly used, but haven't checked.

Reply:
Brewed curl is linked for cargo, so I think this addition is a no-op:
```console
# objdump -p bin/cargo | grep -E "NEEDED|RPATH"
  NEEDED               libz.so.1
  NEEDED               libcurl.so.4
  NEEDED               libssl.so.1.1
  NEEDED               libcrypto.so.1.1
  NEEDED               libgcc_s.so.1
  NEEDED               librt.so.1
  NEEDED               libpthread.so.0
  NEEDED               libm.so.6
  NEEDED               libdl.so.2
  NEEDED               libc.so.6
  NEEDED               ld-linux-x86-64.so.2
  RPATH                /home/linuxbrew/.linuxbrew/Cellar/rust/1.53.0/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/openssl@1.1/lib:/home/linuxbrew/.linuxbrew/opt/zlib/lib:/home/linuxbrew/.linuxbrew/opt/libssh2/lib:/home/linuxbrew/.linuxbrew/opt/brotli/lib:/home/linuxbrew/.linuxbrew/opt/ncurses/lib:/home/linuxbrew/.linuxbrew/opt/readline/lib:/home/linuxbrew/.linuxbrew/opt/libxml2/lib:/home/linuxbrew/.linuxbrew/opt/gettext/lib:/home/linuxbrew/.linuxbrew/opt/libunistring/lib:/home/linuxbrew/.linuxbrew/opt/libidn2/lib:/home/linuxbrew/.linuxbrew/opt/c-ares/lib:/home/linuxbrew/.linuxbrew/opt/jemalloc/lib:/home/linuxbrew/.linuxbrew/opt/libev/lib:/home/linuxbrew/.linuxbrew/opt/nghttp2/lib:/home/linuxbrew/.linuxbrew/opt/util-linux/lib:/home/linuxbrew/.linuxbrew/opt/openldap/lib:/home/linuxbrew/.linuxbrew/opt/rtmpdump/lib:/home/linuxbrew/.linuxbrew/opt/zstd/lib:/home/linuxbrew/.linuxbrew/opt/bison/lib:/home/linuxbrew/.linuxbrew/opt/krb5/lib:/home/linuxbrew/.linuxbrew/opt/curl/lib
```

Reply:
no-op and yet it worked 😄

Reply:
There is a timing/network issue that goes away on repeated re-run, so I don't know if CI passed due to adding curl or just the CI re-run caused by commit.

We probably need to figure out if this is an issue with Homebrew CI setup (GitHub Linux VM and/or Docker container).

Or maybe if this is an issue with brewed curl.

If we can reproduce in separate environment, it may be good to try enabling additional debugs and stacktraces to find out the code path too.